api/src/test/java/org/openmrs/test/jupiter/BaseContextSensitiveTest.java

```diff
@@ -345,19 +349,9 @@ public Properties getRuntimeProperties() {
 		// we don't want to try to load core modules in tests
 		runtimeProperties.setProperty(ModuleConstants.IGNORE_CORE_MODULES_PROPERTY, "true");
 		
-		try {
```

Comment:
@jwnasambu  you are trying to use a variable of a non existing or deleted object.

File tempappdir = File.createTempFile("appdir-for-unit-tests-", ""); was deleted.

Reply:
I believe that's why Travis is throwing   "[ERROR] /home/travis/build/openmrs/openmrs-core/api/src/test/java/org/openmrs/test/BaseContextSensitiveTest.java:[361,33] cannot find symbol

  symbol:   method getAbsolutePath() "

Reply:
@tendomart thanks for the review but I stand to be corrected but I think we need .getRoot()
        .getAbsolutePath()); to sort out the error

Reply:
you can also read https://junit.org/junit5/docs/current/user-guide/#writing-tests-built-in-extensions-TempDirectory ; it also supports the newer java.nio.file.Path on which you can call https://docs.oracle.com/javase/7/docs/api/java/nio/file/Path.html#toAbsolutePath()

I suggest you try using the type java.nio.file.Path.

Reply:
@teleivo thanks let me do it.